tests/Dockerfile.ftest

```diff
@@ -1,3 +1,3 @@
 FROM connectors-base
 
-RUN .venv/bin/pip install -r requirements/ftest.txt
+RUN .venv/bin/pip install ".[ftest]"
```

Comment:
This will install what's under `tests` group _and_ what's in the base dependencies, so more things are being installed than we need at this step. I think that's okay, but if/when we change to `uv` as build-backend we can leverage their command-line to [generate requirements files for a specific dependency group only](https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies).